README.md

```diff
@@ -25,7 +25,7 @@ jobs:
   build:
     runs-on: self-hosted
     steps:
-      - uses: AutoModality/action-clean@v1
+      - uses: freenet-actions/action-clean@v1.1.2
```

Comment:
Ich meine, dass die Konvention für GH-Actions und Versionsnummern ist, dass man bei der Anwendung nur die Major-Version angibt. In diesem Fall also `@v1`.
`v1` zeigt auf den gleichen Commit wie `v1.1.2`.
Deswegen sollte das klappen.

Ich hab mich jetzt auch gefragt, warum wir überhaupt geforkt haben. Musste Hinnerk fragen.
1.1.1 hatte einen Bug, sodass nicht mehr korrekt gelöscht wurde.
Wenn man 1.1.0 verwendet hat, hat sich aber dependabot beschwert, dass es eine neuere Version gab.
Vielleicht sollten wir diesen Hinweis noch mit in die README aufnehmen.

Reply:
Ohh, das mit der Konvention wusste ich nicht 

Ich kann versuchen noch einen Hinweis in der README einzubauen

Reply:
Habe noch was zum Fork geschrieben, gerne nochmal gucken :)